mne/viz/tests/test_3d.py

```diff
@@ -137,11 +137,8 @@ def test_plot_sparse_source_estimates(renderer):
     stc_data[1, 4] = 2.
     vertices = [vertices[inds], np.empty(0, dtype=np.int)]
     stc = SourceEstimate(stc_data, vertices, 1, 1)
-    surf = plot_sparse_source_estimates(sample_src, stc, bgcolor=(1, 1, 1),
-                                        opacity=0.5, high_resolution=False)
-    if renderer.get_3d_backend() == 'mayavi':
-        import mayavi  # noqa: F401 analysis:ignore
-        assert isinstance(surf, mayavi.modules.surface.Surface)
```

Comment:
Shouldn't we keep some variant of this? Can you instead change it to `renderer_interactive.get_3d_backend()`?

Reply:
Indeed that was a bit too agressive, I'll add a branch for mayavi